HerPublicWebsite/appsettings.json

```diff
@@ -4,9 +4,7 @@
     },
     "Logging": {
         "LogLevel": {
-            "Default": "Warning",
-            "Hangfire": "Information",
-            "HerPublicWebsite": "Information"
+            "Default": "Information"

```

Comment:
We could also just omit the entire `"Logging"` block (this is the default anyway) - but it feels nice to be explicit.

Reply:
Agreed :)